examples/01_plotting/plot_surf_stat_map.py

```diff
@@ -0,0 +1,122 @@
+same hemisphere is calculated using Pearson product-moment correlation
+coefficient.
+
+The :func:`nilearn.plotting.plot_surf_stat_map` function is used
+to plot the resulting statistical map on the (inflated) pial surface.
+
+See :ref:`plotting` for more details.
+
+References
+----------
+
+Nooner et al, (2012). The NKI-Rockland Sample: A model for accelerating the
+pace of discovery science in psychiatry. Frontiers in neuroscience 6, 152.
+URL http://dx.doi.org/10.3389/fnins.2012.00152
+
+Dale et al, (1999). Cortical surface-based analysis.I. Segmentation and
+surface reconstruction. Neuroimage 9.
+URL http://dx.doi.org/10.1006/nimg.1998.0395
+
+Fischl et al, (1999). Cortical surface-based analysis. II: Inflation,
+flattening, and a surface-based coordinate system. Neuroimage 9.
+http://dx.doi.org/10.1006/nimg.1998.0396
+
+Destrieux et al, (2010). Automatic parcellation of human cortical gyri and
+sulci using standard anatomical nomenclature. NeuroImage, 53, 1.
+URL http://dx.doi.org/10.1016/j.neuroimage.2010.06.010.
+"""
+
+###############################################################################
+from nilearn import plotting
+from nilearn import datasets
+from scipy import stats
+import numpy as np
+
+###############################################################################
+# Retrieve the data
+nki_dataset = datasets.fetch_surf_nki_enhanced(n_subjects=1)
+
+# NKI resting state data set of one subject left hemisphere in fsaverage5 space
+resting_state = nki_dataset['func_left'][0]
+
+# Destrieux parcellation left hemisphere in fsaverage5 space
+destrieux_atlas = datasets.fetch_atlas_surf_destrieux()
+parcellation = destrieux_atlas['map_left']
+labels = destrieux_atlas['labels']
+
+# Retrieve fsaverage data
+fsaverage = datasets.fetch_surf_fsaverage5()
+
+# Fsaverage5 left hemisphere surface mesh files
+fsaverage5_pial = fsaverage['pial_left']
+fsaverage5_inflated = fsaverage['infl_left']
+sulcal_depth_map = fsaverage['sulc_left']
+
+###############################################################################
+# Load resting state time series and parcellation
+timeseries = plotting.surf_plotting.load_surf_data(resting_state)
+
+# Extract seed region: dorsal posterior cingulate gyrus
+pcc_region = 'G_cingul-Post-dorsal'
+pcc_labels = np.where(parcellation == labels.index(pcc_region))[0]
+
+# Extract time series from seed region
+seed_timeseries = np.mean(timeseries[pcc_labels], axis=0)
+
+# Calculate Pearson product-moment correlation coefficient between seed
+# time series and timeseries of all cortical nodes of the hemisphere
+stat_map = np.zeros(timeseries.shape[0])
+for i in range(timeseries.shape[0]):
+    stat_map[i] = stats.pearsonr(seed_timeseries, timeseries[i])[0]
+
+# Re-mask previously masked nodes (medial wall)
+stat_map[np.where(np.mean(timeseries, axis=1) == 0)] = 0
+
+###############################################################################
+# Display ROI on surface
```

Comment:
It might be useful to split the cell below is several smaller cells: it's easier to comprehend.